src/kernel/entry.c

```diff
@@ -20,8 +21,13 @@ void start() {
 	// Init memory system and make room for the kernel
 	mem_init(0xFFFFF);
 
-//	kprintf("\e[2J%% Hello \e[1;33m%s\e[m wo%cld, party like it's \e[1m%#0.8x\e[m again!\n", "sweet", 'r', -1920);
-
+    kprintf("\e[2J%% Hello \e[1;33m%s\e[m wo%cld, party like it's \e[1m%#0.8x\e[m again!\n", "sweet", 'r', -1920);
+    void* ptr = kmalloc(50000000);
+    kprintf("\n%d", ptr);
+    void* ptr2 = kmalloc(1000);
+    kprintf("\n%d", ptr2);
+    kprintf("\n%d", kmsz(ptr));
+    halt();
```

Comment:
don't `halt()` in the middle of kernel init please

Reply:
I deleted the entire code that is meant for testing the new feature.